cluster-autoscaler/version/version.go

```diff
@@ -17,4 +17,4 @@ limitations under the License.
 package version
 
 // ClusterAutoscalerVersion contains version of CA.
-const ClusterAutoscalerVersion = "1.25.1"
+const ClusterAutoscalerVersion = "1.25.2"
```

Comment:
No need to change this, we'll do it when we cut the release.

Reply:
Thanks for the review. I will amend this bit.